src/Server.php

```diff
@@ -90,6 +90,25 @@ public function startBrokerSession()
 
         $this->brokerId = $this->validateBrokerSessionId($sid);
     }
+		
+	/**
+	 * Get session ID from header Authorization or from $_GET/$_POST
+	 */
+	protected function getBrokerSessionID(){
+		$headers = getallheaders();
```

Comment:
Please adhere to [PSR2](http://www.php-fig.org/psr/psr-2/) and use 4 spaces, not tabs.